styles.css

```diff
@@ -193,6 +184,12 @@ img {
   font-size: 14px;
 }
 
+hr {
```

Comment:
I see now(from my question before about the <hr> tag use. Is this used to style the page, like adding spacing? If so, was there a reason you used this element rather than just a <div>?